### ISSUE TITLE
Using same-page anchors as links inside the generated document

### ISSUE THREAD

**Issue by techquestions** [Aymkdn/html-to-pdfmake#115]
It would be nice to convert `<a href="#id-of-element-on-the-page">Click me</a>` to [links inside a generated PDF document](https://pdfmake.github.io/docs/0.1/document-definition-object/links/). 

Here is a code that can be used to demonstrate what I mean in the [PDFmake playground](http://pdfmake.org/playground.html) (the key parts are `"linkToDestination": "test"` and `"id": "test"`):

```
var dd = {
  "content": [
    {
      "text": "test",
      "color": "blue",
      "decoration": [
        "underline"
      ],
      "style": [
        "html-a"
      ],
      "linkToDestination": "test",
      "nodeName": "A"
    },
    {
      "text": "test",
      "id": "test",
      "pageBreak": 'before',
      "nodeName": "P",
      "margin": [
        0,
        5,
        0,
        10
      ],
      "style": [
        "html-p"
      ]
    }
  ],
}
```

Would you be open for such change?

**Comment by Aymkdn** [comment]
> Would you be open for such change?

Yes. It should not be too complex. I can look at it today.

**Comment by Aymkdn** [comment]
I've just published v2.3.3

Here is an example:
```html
<a href="#test">Go to test</a>

<div class="pdf-pagebreak-before">
  <h1 id="test">Here you go!</h1>
</div>
```

**Comment by techquestions** [comment]
Thank you!
